README.md

```diff
@@ -160,6 +162,94 @@ Below is where to find these environment variables:
 * `/ping` to check if the bot is online
 
 
+## Using the Leaderboard
+
+The main purpose of the Discord bot is to allow servers to host coding competitions through Discord. 
+The leaderboard was designed for evaluating GPU kernels, but can be adapted easily for other
+competitions. The rest of this section will mostly refer to leaderboard submissions in the context
+of our GPU Kernel competition.
+
+
+> [!NOTE]
+> All leaderboard commands have the prefix `/leaderboard`, and center around creating, submitting to,
+> and viewing leaderboard statistics and information. 
+
+### Creating a new Leaderboard
+![Leaderboard creation command](assets/img/lb_creation.png)
```

Comment:
Personally would remove this, in favour of only keeping the textual description, same as on the other commands. However, up to discuss this

Reply:
Oh sure, I'll delete. This was also something I was unsure about, but I'll just remove it for now.